### ISSUE TITLE
confusing code in ProbeNode.js

### ISSUE THREAD

**Issue by pixelzoom** [phetsims/scenery-phet#665]
Noted while working on https://github.com/phetsims/scenery-phet/issues/488. 

There's some confusing code in ProbeNode.js.  Two primary things:

(1) "sensor" is defined as "circular part of the ProbeNode is called the sensor".  But then there are functions and variables like `createSensorShape` and `sensorShape` which are actually the entire probe, not the sensor.

(2) Function `createSensorShape` does not close the Shape, and relies on clients to do that in 2 places, shown below. This is confusing and error prone.  `createSensorShape` should close its Shape.

```js
    const sensorShape = createSensorShape()
      .moveTo( innerRadius, 0 )
      .arc( 0, 0, innerRadius, Math.PI * 2, 0, true )
      .close();
...
      const outline = createSensorShape().close();
```

I'm going to clean this up, so we're in a better place to address #196.

**Comment by pixelzoom** [comment]
Done in the above commits. Tested in scenery-phet demo and in a bunch of sims.

@samreid would you mind taking a peek, since you're the primary author, and you and I have collaborated on ProbeNode in the past?

**Comment by pixelzoom** [comment]
Oops.  After this change, the probe is no longer draggable by the sensor area.  Back to me.

**Comment by pixelzoom** [comment]
Oops again. There's nothing wrong with these changes. The sensor was not draggable due to a local change that I made, experimenting with a possible  fix for #488.

**Comment by samreid** [comment]
I reviewed the commits and tested in the scenery-phet demo and did not have any recommendations.  I think this issue can be closed, but back to @pixelzoom just in case there's more to do first.
